app/pages/patient/patientinfo.js

```diff
@@ -374,12 +391,12 @@ var PatientInfo = React.createClass({
     }
 
     var birthday = formValues.birthday;
-    if (!(birthday && datetimeUtils.isValidDate(birthday))) {
+    if (!(birthday && sundial.isValidDate(birthday))) {
```

Comment:
with the updates in sundial 

![screen shot 2015-03-09 at 1 19 29 pm](https://cloud.githubusercontent.com/assets/5168993/6548489/227c678e-c65f-11e4-9893-4101d7187678.png)

now fails validation